Automattic/crowdsignal-forms
# Add Feedback API functionality

This PR depends on #75 

This PR adds all the backend files, methods and registration for the new Feedback block to work against the API `/feedback` endpoint.

## Test instructions
Checkout and `make client`.

  - Add a Feedback block to a post and publish it
  - Visit your Crowdsignal dashboard to see the newly created survey is shown
  - Visit the post where you published the Feedback block, submit some feedback
  - Visit the results page and check your submitted feedback can be seen
